engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -24,7 +24,7 @@ sealed trait JobDescriptor[K <: CallKey] {
   def locallyQualifiedInputs: CallInputs
 }
 
-final case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
+case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
```

Comment:
@mcovarr Wouldn't the answer to that be to bring an alternate implicit into scope?